src/main/java/com/solvd/navigator/model/util/AllPairShortestPath.java

```diff
@@ -0,0 +1,91 @@
+package com.solvd.navigator.model.util;
+
+import java.io.*;
+import java.lang.*;
+import java.util.*;
+
+public class AllPairShortestPath {
+    public final static int INF = 99999, V = 4;
```

Comment:
In our case variable V should equal to 7 cause we have 7 nodes right?

Reply:
That's right. To keep in mind that if he change that, he should also change the graph matrix.

Reply:
yes it should be 7. I didn't change it because I didn't have our 7-nodes example yet. I'm going to change it and make a new push